benchmarks/mmmu/mmmu.py

```diff
@@ -0,0 +1,169 @@
+import ast
+import base64
+from io import BytesIO
+
+from datasets import concatenate_datasets, load_dataset
+from PIL import Image
+
+from inspect_ai import Task, task
+from inspect_ai.dataset import Sample
+from inspect_ai.dataset._dataset import MemoryDataset
+from inspect_ai.model import ChatMessageUser, ContentImage, ContentText, GenerateConfig
+from inspect_ai.scorer import choice, model_graded_fact
+from inspect_ai.solver import generate, multiple_choice, prompt_template
+
+
+# Helper function to convert images to data URLs
```

Comment:
since you have a lot of functions in here. Maybe move some to a separate helper file?

Reply:
Agreed, thanks! Done